### PR TITLE
Mesh tools

### PR DESCRIPTION
**Pull request by bpatchasaheb**
Mesh Tools have been added.
I set bunny icon for mesh tools.
You can change the icon later if you want.

Create Controller(world space) viewport pause fix.

### CODE REVIEW

File: python/mmSolver/tools/holdoutsdisableallmeshes/tool.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+from __future__ import print_function
+
+import maya.cmds as cmds
+import mmSolver.logger
+import mmSolver.tools.holdoutsenableselectedmeshes.lib as lib
+
+LOG = mmSolver.logger.get_logger()
+
+
+def main():
```
Comment (david-cattermole) [inline python/mmSolver/tools/holdoutsdisableallmeshes/tool.py:33]: I think this entire tool can be combined into a single tool, and we break the "one tool, one function" rule.
The tool is so small I don't think we need separate tools for each, just put them all under a tool named something like "setmeshholdouts".

So we'd have functions like:
```
setmeshholdouts.tool.enable_all_meshes()
setmeshholdouts.tool.disable_all_meshes()
setmeshholdouts.tool.enable_selected_meshes()
setmeshholdouts.tool.disable_selected_meshes()
```

**Review by david-cattermole — CHANGES_REQUESTED** [review]
This is fantastic @bpatchasaheb!!!

The bunny icon is cool as a first pass too, I like it.

I've suggested a few small changes.

The only really important change is making sure the copyright notice changes for the delaunator.py file.

File: share/config/shelf.json [hunk 2]
```diff
@@ -92,6 +92,13 @@
             "zdepth_tools/zdepth_popup/camera_object_scale_remove",
             "zdepth_tools/zdepth_popup/---",
             "zdepth_tools/zdepth_popup/screen_z_transform_bake",
+            "mesh_tools/mesh_tools_popup/---Create",
+            "mesh_tools/mesh_tools_popup/mesh_from_locators",
+            "mesh_tools/mesh_tools_popup/---Hold-Outs",
```
[the remark this answers is not in the document]
Comment (bpatchasaheb) [reply share/config/shelf.json:97]: That makes sense, I would put them under "Display tools" menu.

File: python/mmSolver/tools/holdoutsdisableallmeshes/tool.py (hunk 1 above)
Comment (bpatchasaheb) [reply python/mmSolver/tools/holdoutsdisableallmeshes/tool.py:33]: I would make changes accordingly. Thanks.

**Review by david-cattermole — APPROVED** [review]
This looks great, thanks @bpatchasaheb!

**Pull request merged**